services/embedder/tests/test_embedder_grpc.py

```
@@ -0,0 +1,266 @@
+
+
+class StubEmbedder:
+    """Deterministic embedder for wire-contract tests."""
+
+    device = "cpu"
+    EMBEDDING_DIM = 2048
+
+    def __init__(self, _config):
+        pass
+
+    def embed_texts(self, texts, task="retrieval.passage", batch_size=None):
```

Ruff (0.15.10)

[warning] Unused method argument: `batch_size`

(ARG002)